packages/ozone/src/tag-service/embed-content-type-tagger.ts

```diff
@@ -0,0 +1,45 @@
+import { AppBskyEmbedImages } from '@atproto/api'
+import { langLogger as log } from '../logger'
+import { ContentTagger } from './content-tagger'
+
+export class EmbedContentTypeTagger extends ContentTagger {
+  tagPrefix = 'embed-content-type:'
+  isApplicable(): boolean {
+    return Boolean(
+      this.subjectStatus &&
+        this.subject.isRecord() &&
+        this.subject.uri.includes('app.bsky.feed.post') &&
```

Comment:
It would be cozier if we either checked `includes('/app.bsky.feed.post/')`, or went ahead and parsed the URI and checked its collection.

```ts
const getCollection(uri: string) {
  return new AtUri(uri).collection
}
```